.github/linters/.yaml-lint.yml

```
@@ -0,0 +1,34 @@
+  line-length:
+    max: 120
+    level: warning # Consider changing this to 100 for better readability
+
+  new-line-at-end-of-file:
+    level: warning
+
+  trailing-spaces:
+    level: error # Enforce no trailing spaces
+
+  brackets:
+    level: error # Enforce proper bracket usage
```

yamllint

[warning] too few spaces before comment

(comments)